tests/conftest.py

```diff
@@ -31,6 +31,9 @@
 pd.options.mode.chained_assignment = "raise"
 
 
+pytest.register_assert_rewrite("kartothek.io.testing")
```

Comment:
I don't really get the docs of `pytest.register_assert_rewrite`. What I suspect: Will this make the output better readable?

Reply:
Yes. For example:

```python
assert some_dict == other_dict
```

in lib dir (not within `tests/`) only raises an `AssertionError` so the tests fails (this is correct), but you don't get a visual diff between the two dicts. This is only possible if pytest can rewrite the assertion (basically it replaces the bytecode snipped) to hook into it. See https://docs.pytest.org/en/latest/writing_plugins.html#assertion-rewriting

I find it extremely helpful to get the full pytest feature set (aka visual diffs) in our test suite.

Reply:
good to know, thanks!